Examples/Resources/Extractors/SAT-features-competition2012_revised_without_SatELite_sparkle/sparkle_extractor_wrapper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""Template for users to create Feature Extractor wrappers."""
+
+import sys
+import os
+import time
+import random
+import argparse
+import subprocess
+from pathlib import Path
+
+global sparkle_special_string
+sparkle_special_string = r'__@@SPARKLE@@__'
+
+def get_last_level_directory_name(filepath):
```

Comment:
Remove and use Path objects instead

Reply:
What do you mean? We use paths, but for some cases only the basename is required

Reply:
last level directory is file.parent (Path.isfile) or directory.name (! Path.isfile)